### PR TITLE
added token to podcast route, added s3o to non-podcast routes

### PR DESCRIPTION
**Pull request by railsagainstignorance**
be gentle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,11 @@ More details
 
 # new end points
 
-* /podcast?rss=URL_OF_ARTICLE_RSS_FEED
+* /podcast?token=TOKEN&voice=VOICE_ID&rss=URL_OF_ARTICLE_RSS_FEED
    * return full podcast feed, including refs to mp3 files
-* /mp3?id=ID_OF_MP3_CONTENT
+   * VOICE_ID needs to match one of the known voices
+   * TOKEN needs to match the value of env param: PODCAST_TOKEN
```
Comment (seanmtracey) [inline README.md]: Where do I set the envvar podcast token?

File: bin/lib/formatContentForReading.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const REMOVALS = [
 ];
 const REMOVALS_PATTERN = `\\b(${REMOVALS.join('|') })\\b`;
 const REMOVALS_REGEXP = new RegExp(REMOVALS_PATTERN, 'ig');
-debug(`REMOVALS_REGEXP=${REMOVALS_REGEXP}`);
+// debug(`REMOVALS_REGEXP=${REMOVALS_REGEXP}`);
```
Comment (seanmtracey) [inline bin/lib/formatContentForReading.js]: If the debugs are no longer being used, can we remove them? Or should we leave them on for the prod server?

File: index.js [hunk 3]
```diff
@@ -7,40 +7,66 @@ const app = express();
...
+    autovoice.podcast(rssUrl, voice)
+    .then(feed => {
+      res.set('Content-Type', 'application/rss+xml');
+      res.send(feed);
+    })
+    ;
+  }
 });
 
 app.get('/audio.mp3', (req, res) => {
```
Comment (seanmtracey) [inline index.js]: Does this endpoint not also trigger a generation of audio content? Does it not require a token check too?

Comment (railsagainstignorance) [inline index.js]: pulls it from cache, else "no mp3 content for fileId="
so, no, does not generate content

File: bin/lib/formatContentForReading.js (hunk 2 above)
Comment (railsagainstignorance) [inline bin/lib/formatContentForReading.js]: hm, ok

Comment (Lily2point0) [inline bin/lib/formatContentForReading.js]: +1 on this comment

File: index.js [hunk 4]
```diff
@@ -7,40 +7,66 @@ const app = express();
...
-});
+// these routes do *not* have s3o
+
+app.use('/static', express.static('static'));
 
 app.get('/__gtg', (req, res) => {
 	res.status(200).end();
 });
 
+const PODCAST_TOKEN = process.env.PODCAST_TOKEN;
```
Comment (Lily2point0) [inline index.js]: Might be better as a try/catch in the `/podcast` route?

File: README.md (hunk 1 above)
Comment (railsagainstignorance) [inline README.md]: reworded README

File: bin/lib/formatContentForReading.js (hunk 2 above)
Comment (railsagainstignorance) [inline bin/lib/formatContentForReading.js]: done

File: index.js (hunk 4 above)
Comment (railsagainstignorance) [inline index.js]: where it is will cause the app to fail on startup, which hopefully means heroku will not deploy it

File: index.js (hunk 3 above)
Comment (seanmtracey) [inline index.js]: 👍

File: README.md (hunk 1 above)
Comment (seanmtracey) [inline README.md]: 👍

**Pull request merged**